frontend/src/static/js/v_authorship.js

```diff
@@ -201,6 +207,18 @@ window.vAuthorship = {
       this.activeFilesCount = this.selectedFiles.length;
     },
 
+    getFontColor(color) {
```

Comment:
This actually overlaps with #1025. Perhaps wait for that PR to be merged first?

Reply:
not very sure if it is worthy to add a global function

Reply:
Perhaps in a util class?

Reply:
can but should that be a seperate pr? to move all global func into util.js